```
/tmp/nix-build-libclipboard.drv-0/source/src/clipboard_cocoa.c:17:10: fatal error: 'Cocoa/Cocoa.h' file not found
#include <Cocoa/Cocoa.h>
         ^~~~~~~~~~~~~~~
1 error generated.
```

@wegank could you help with this 🥹 Looks like `darwin.apple_sdk.frameworks.Foundation`

--- a/pkgs/by-name/li/libclipboard/package.nix
+++ b/pkgs/by-name/li/libclipboard/package.nix
@@ -1,0 +1,38 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  libxcb,
+  libXau,
+  libXdmcp,
+  darwin,
+  lib
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  name = "libclipboard";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "jtanx";
+    repo = "libclipboard";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-553hNG8QUlt/Aff9EKYr6w279ELr+2MX7nh1SKIklhA=";
+  };
+
+  buildInputs = [ libxcb libXau libXdmcp ]
+    ++ lib.optional stdenv.isDarwin [ darwin.apple_sdk.frameworks.Cocoa ];
+  nativeBuildInputs = [ cmake pkg-config ];
+
+  cmakeFlags = [ "-DBUILD_SHARED_LIBS=ON" ];
+  outputs = [ "out" "dev" ];
+
+  meta = {
+    description = "Lightweight cross-platform clipboard library";
+    homepage = "https://jtanx.github.io/libclipboard";
+    platforms = lib.platforms.unix;
+    license = lib.licenses.mit;
+    maintainers = [ lib.maintainers.sigmanificient ];
+  };
+})

